### ISSUE TITLE
Color of text not changeable

### ISSUE THREAD

**Issue by subnut** [badges/shields#5455]
:clipboard: **Description**
There is no option that I could find at [shields.io](https://shields.io/#styles) that allows to set the background color. This is important in cases where you might want the badge color to be a light one. The default white is not very visible in that case. Or say, someone wants to make the labelColor white!

If you believe that it is unreasonable to expect that, look at this:
![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=white)
![Linux](https://img.shields.io/badge/Linux-FCC624?style=for-the-badge&logo=linux&logoColor=black)

You yourself judge which penguin looks better. Then look at the text to see if it is as readable as the text of the IDE you love :wink:
Don't you think the text would have been more readable only if the text was _BLACK_ ?

Oh, why I chose _that_ particular color? It was suggested by [simpleicons.org](https://simpleicons.org/?q=linux) for this logo
Which maniac would choose to do this? [ME!](https://subnut.github.io/ncm2-github-emoji/) :sweat_smile: 

<br>

#### TL;DR
Add a new `labelTextColor` attribute (for the left part) and `textColor` (for the right part)

**Comment by subnut** [comment]
> I believe you are correct.

:disappointed: 

> Will not commit to it right now, but I'll keep it in mind

Thanks very much! :smile:

> No worries, though if it's okay with you I suggest we close this issue and leave the others open. :)

Of course! :innocent:

**Comment by ooliver1** [comment]
![image](https://user-images.githubusercontent.com/34910574/198157280-6b011c07-d9dd-43f3-965d-09f9f2837933.png)
I don't know if its just me but this is like, pretty hard to read, at least on my monitor and bad eyes
